csrc/scheduler/vectorize_helper.cpp

```diff
@@ -800,13 +800,33 @@ Val* ContiguousInnerDimensionsMapper::getContigMergeOfInnerSize(
       }
     }
 
+    // Get the logical ID corresponding to the allocation ID.
+    auto exprs = DependencyCheck::getAllExprsBetween(
```

Comment:
The `projectAllocationToLogical` can be made to return a pair. At the moment, this seemed easier to read to me. Open to modifying the utility function in the interest of merging too.